examples/src/protobuf/org/pantsbuild/example/service/BUILD

```diff
@@ -0,0 +1,8 @@
+# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Please update this header, and make sure that new files you've added have up to date headers. Sorry that we don't have automation for this =)

Reply:
Header updated 👍